bot/discord/commands/server/docs.js

```diff
@@ -0,0 +1,74 @@
+exports.run = async(client, message, args) => {
+
+    let helpEmbed = new Discord.MessageEmbed()
+        .setColor("RED").setDescription(`List of servers:\n`)
+        //.addField("__**Minecraft:**__", "Forge \nPaper \nBedrock \nPocketmineMP \nWaterfall \nSpigot", true)
+        //.addField("__**Grand Theft Auto:**__", "alt:V \nmultitheftauto \nRage.MP \nSA-MP", true)
+        .addField("__**Bots:**__", "NodeJS \nPython \nJava \naio \nRedDiscordBot", true)
+        //.addField("__**Source Engine:**__", "GMod \nCS:GO \nARK:SE", true)
+        .addField("__**Voice Servers:**__", "TS3 \nMumble", true)
+        //.addField("__**SteamCMD:**__", "Rust \nDaystodie \nArma \nAssettocorsa \nAvorion \nBarotrauma", true)
+        .addField("__**Databases:**__", "MongoDB \nRedis \nPostgres", true)
+        .addField("__**WebHosting:**__", "Nginx", true)
+        .addField("__**Custom Eggs:**__", "ShareX", true)
+        .addField("__**Software:**__", "codeserver \ngitea \nhaste", true)
+        //.addField("__**Storage:**__", "storage", true)
+        .setFooter("Example: DBH!server docs NodeJS")
+
+
+    if (!args[1]) {
+        await message.channel.send(helpEmbed);
+        return;
+    };
+
+    if(args[1] == "list") {
+        await message.channel.send(helpEmbed);
+        return;
+    }
+
+    const embed = new Discord.MessageEmbed();
+    switch(args[1].toLowerCase()) {
+       case "aio":
+         embed.setDescription("`All In One`, short `aio` is a unique server type that contains several features that distinguish it from other Development(Bot & Codeserver) server types. This type brings all of the other types in a 1 single type and lets user interact with console. [Source: docs.dbh.wtf](https://docs.dbh.wtf)\nRecommend for beginners, as it is easy to use.\n**Do not use aio to host gameservers. It is not allowed and your server will be deleted.**");
```

Comment:
I don't think having 3rd party docs in the bot is a good idea imo

Reply:
then there's no point in this command as each description is partially takes from the community docs and its not good to not mentioning the source

Reply:
Then yea the command is not needed. The staff have voted in staff chat to count it as advertising anyways :thinking: